lib/chef/knife/ec2_server_create.rb

```diff
@@ -67,10 +67,18 @@ class Ec2ServerCreate < Knife
         :proc => Proc.new { |groups| groups.split(',') }
 
       option :security_group_ids,
-        :short => "-g 'X,Y,Z'",
-        :long => "--security-group-ids 'X,Y,Z'",
-        :description => "The security group ids for this server; required when using VPC,Please provide values in format --security-group-ids 'X,Y,Z'",
-        :proc => Proc.new { |security_group_ids| security_group_ids.split(',') }
+        :short => "-g SECURITY_GROUP_IDS",
+        :long => "--security-group-ids",
```

Comment:
Yes @mwrock.

@Vasu1105: Keep whatever behavior we've added for `--security-group-ids` but have it print a warning that you should switch to using `--security-group-id` and specify a single group with each use rather than a list, and that the `--security-group-ids` option will be removed in a future release. the `-g` short flag should go with the new `--security-group-id`.